examples/protocol_specification_ex/oef_search.yaml

```diff
@@ -37,3 +37,15 @@ ct:OefErrorOperation: |
       }
   OefErrorEnum oef_error = 1;
 ...
+---
+entry: [register_service, unregister_service, search_services]
```

Comment:
I would make it so `unregister` is a reply to `register` and avoid the second type of dialogue sequence. Then we would only have two sequences:
`register`, (`error`), `unregister`, (`error`)
`search`, `results`/`error`

Reply:
Shouldn't be a problem replying to your own message. See below.